README.md

```diff
@@ -12,6 +12,13 @@ The derive macro can be used on tuple or field structs to produce a new struct r
 
 ```rust
 #[derive(Debug, Default, PartialEq, Diff)]
+// this will apply the specified derives on the generated 'ProjectMetaDiff' struct
+#[diff(attr(
+    #[derive(Debug, PartialEq)]
+))]
+struct MyStruct {
```

Comment:
Let's remove the `MyStruct`, since the attribute is supposed to apply to `ProjectMeta`